### PR TITLE
update docs from beats

### PR DESCRIPTION
**Pull request by jalvz**

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -46,6 +46,8 @@ include::./copied-from-beats/shared-securing-beat.asciidoc[]
 
 include::./copied-from-beats/monitoring/monitoring-beats.asciidoc[]
 
+include::./copied-from-beats/processors-using.asciidoc[]
```
Comment (graphaelli) [inline docs/index.asciidoc]: do we [want to] support processors?

File: docs/copied-from-beats/loggingconfig.asciidoc [hunk 2]
```diff
@@ -155,6 +155,15 @@ Examples:
...
 associated with the file, as well as read access to all other users.
 
+[float]
+==== `logging.files.interval`
+
+Enable log file rotation on time intervals in addition to size-based rotation.
+Intervals must be at least 1s. Values of 1m, 1h, 24h, 7*24h, 30*24h, and 365*24h
+are boundary-aligned with minutes, hours, days, weeks, months, and years as
+reported by the local system clock. All other intervals are calculated from the
+unix epoch. Defaults to disabled.
```
Comment (simitt) [inline docs/copied-from-beats/loggingconfig.asciidoc]: When documenting we should also add this option to `apm-server.yml`.

File: docs/copied-from-beats/outputconfig.asciidoc [hunk 3]
```diff
@@ -215,54 +213,112 @@ for more information about the environment variables.
...
-+"{beatname_lc}-{version}-2017.04.26"+). If you change this setting, you also
++"{beatname_lc}-{version}-{localdate}"+). If you change this setting, you also
 need to configure the `setup.template.name` and `setup.template.pattern` options
-(see <<configuration-template>>).
-
-When loading the pre-built Kibana dashboards via {beatname_uc},
-you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
+(see <<configuration-template>>). If you are using the pre-built Kibana
+dashboards, you also need to set the `setup.dashboards.index` option (see
+<<configuration-dashboards>>).
```
Comment (simitt) [inline docs/copied-from-beats/outputconfig.asciidoc]: This removes the clause again _When loading the pre-built Kibana dashboards via APM Server_ ... which should not be removed, as it is deprecated.

File: docs/copied-from-beats/processors-using.asciidoc [hunk 4]
```diff
@@ -0,0 +1,1124 @@
+[[defining-processors]]
```
[the remark this answers is not in the document]
Comment (simitt) [reply docs/copied-from-beats/processors-using.asciidoc]: I'd not actively document this atm.

File: docs/copied-from-beats/shared-kibana-config.asciidoc [hunk 5]
```diff
@@ -109,8 +109,7 @@ Example configuration:
 
 [source,yaml]
 ----
-setup.kibana.host: "192.0.2.255:5601"
-setup.kibana.protocol: "https"
+setup.kibana.host: "https://192.0.2.255:5601"
```
Comment (simitt) [inline docs/copied-from-beats/shared-kibana-config.asciidoc]: I am confused now with `http` and `https` usage. I know you haven't written that, but why did that change and why do we have a mix? I would not pull this in.

File: docs/copied-from-beats/shared-logstash-config.asciidoc [hunk 6]
```diff
@@ -14,7 +14,7 @@
 To send events to {ls}, you also need to create a {ls} configuration pipeline
 that listens for incoming Beats connections and indexes the received events into
 {es}. For more information, see the section about
-{stack-ov}/get-started-elastic-stack.html#logstash-setup[configuring {ls}] in
+{stack-gs}/get-started-elastic-stack.html#logstash-setup[configuring {ls}] in
```
Comment (simitt) [inline docs/copied-from-beats/shared-logstash-config.asciidoc]: We don't have a `stack-gs` variable defined.

File: docs/copied-from-beats/processors-using.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (jalvz) [reply docs/copied-from-beats/processors-using.asciidoc]: sounds good, will remove 👍

**Pull request closed without merging**